### ISSUE TITLE
Spare the Balloon circumvented

### ISSUE THREAD

**Issue by torso** [the-butterfly-effect/tbe#304]
Rotate the bottle clockwise and it will fit to the left of the ball. (The balloon will not have enough time to pop.) Alternatively, just push the ball with the wedge.

Don't know the expected solution, but it shouldn't it at least require a basic effort to save the balloon?
![spare the balloon](https://cloud.githubusercontent.com/assets/748702/16488996/ae91a598-3ed3-11e6-9e33-9d3299270bbc.jpg)

**Comment by Wuzzy2** [comment]
Yes, this level has many issues, this is already mentioned in The Second Big Level Polishing, so your issue is kind of a duplicate, but I keep it open because of the screenshot.
I have once tried to tweak it in a way that the nasty stuff doesn't fit into the right half without using chickenwire, but I failed; I always found another stupid shortcut.
You could also place the cola bottle over the ramp with the bottle cap pointing to the soccer ball.

@kaa-ching, what about just hiding the entire right side behind chickenwire? Or would that make the level too easy?

**Comment by kaa-ching** [comment]
@Wuzzy2 I understand you reluctance to put chicken wire over all of the right-side.
However, in this case, in my view: by hinting where the solution is not, you're only a small step closer to a real solution.

**Comment by kaa-ching** [comment]
Indeed, this level remains to have holes :-1: :unamused: 

![more-304-trouble](https://cloud.githubusercontent.com/assets/4249656/16586134/8ff3eb88-42c4-11e6-9be7-173056b74a7e.png)

**Comment by kaa-ching** [comment]
commit f041534 helps a bit by plugging *that* specific hole.

**Comment by kaa-ching** [comment]
@torso @Wuzzy2 : another holes that need plugging?

**Comment by Wuzzy2** [comment]
Yes. I found two.

Number 1:
https://i.imgur.com/Scsxr0T.png

Number 2 (similar):
Using cactus and a tilted floor, you can also push the first domino. This is even possible with a flat floor.

Suggestions:
* Remove cactus
* Add weight as a distraction
* Fix rotation of cola+mint bottle

**Comment by Wuzzy2** [comment]
After experimenting for a while, I think it's hard to fix the repaining holes without changing the core idea of this level. I ran out of ideas.

The remaining alternative solutions aren't that primitive anymore.

Maybe we just leave that level as-is.
